### ISSUE TITLE
Grid labels outside of the plot

### ISSUE THREAD

**Issue by craggoar** [pytroll/satpy#1704]
As far as I know, it is currently not possible to put the grid labels outside of MSG plots in the png/ geotiff format. I miss that feature for the reason that the labels would be more easy to read while the black/ white color style can be preserved. Furthermore this would make the plots appear more professional.

I would like to have the option to choose whether the lables are inside or outside of the plot.

**Additional context**
As alternative I considered plotting an extended overlay without data, since this would also result in a better readability.

**Comment by mraspaud** [comment]
I totally agree, this would make for nicer plots. Maybe not geotiff (would it make sense to geolocate the white space around?), but definitely the other image formats. Any help in starting a pr with this would be highly appreciated. Although it would probably need to go in the pycoast package.

**Comment by anikfal** [comment]
Hi,
According to your concern, you want the labels (60E, 70E, ...) to be outside of the plot (the image below).
It seems to be a pycoast issue, without any relevance to satpy. Am I right? @craggoar @mraspaud 
![sample_msg](https://user-images.githubusercontent.com/11738727/148765231-ccf8ab31-5b0e-4641-8984-6d15fe2dd450.png)
